### PR TITLE
SDS Local ports conflicts CIR-177

### PR DESCRIPTION
**Pull request by SriHV**
### Motivation and Context
This is in line with the changes required as per the [CIR-177](https://jira.ons.gov.uk/browse/CIR-177) Local Ports - Conflicts. The PR for CIR can found in this [link](https://github.com/ONSdigital/eq-cir-fastapi/pull/23)

### What has changed
*  `FIRESTORE EMULATOR UI` runs at host port `4001`
*  The FastAPI application runs at port `0.0.0.0:3033` 

### How to test?
Run CIR and SDS docker containers at same time and check if there are any issues

### CODE REVIEW

**Review by IsaiahLai111 — COMMENTED** [review]
Looks good. Can you also change devtools>firebase.json as well? Thanks

**Review by tidmanmike — APPROVED** [review]

**Comment by SriHV** [comment]
> Looks good. Can you also change devtools>firebase.json as well? Thanks And also the readme file and Makefile

Thanks, I just tried changing ports firebase.json and it threw error. Should I change it anywhere else as well?

**Review by Mohammed1804 — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ services:
     image: spine3/firebase-emulator
     ports:
       - 8080:8080
-      - 4000:4000
+      - 4001:4000
```
Comment (IsaiahLai111) [inline docker-compose.yml:47]: Change both side to 4001 as http://0.0.0.0:4001/ cannot be reached now

**Review by IsaiahLai111 — COMMENTED** [review]
Please also change the local port in Makefile as currently local integration test is failing

**Review by IsaiahLai111 — APPROVED** [review]

**Pull request merged**